pkg/database/realm.go

```diff
@@ -31,6 +32,11 @@ type Realm struct {
 	// Name is the name of the realm.
 	Name string `gorm:"type:varchar(200);unique_index"`
 
+	// DisableLikelyReports and DisableNegativeReports are realm-specific
+	// configurations to change how codes are issued by the realm.
+	DisableLikelyReports   bool `gorm:"type:boolean; not null; default:false"`
```

Comment:
This weird inversion is required because Gorm won't set values for default values, and the default value for a boolean is false.